### PR TITLE
Add release notes for 14.3

### PR DESCRIPTION
**Pull request by tas50**
Signed-off-by: Tim Smith <tsmith@chef.io>

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,9 +1,109 @@
...
+
+#### Properties
+
+- `modname` - The name of the kernel module.
+- `load_dir` - The directory to load modules from.
+- `unload_dir` - The modprobe.d directory.
+
+### ssh_known_hosts_entry
+
+Use the ssh_known_hosts_entry resource to append an entry for the specified host in /etc/ssh/ssh_known_hosts or a user's known hosts file if specified.
```
Comment (stuartpreston) [inline RELEASE_NOTES.md]: Append an entry -> Add an entry?  Append would imply an entry was appended on every run but I'm guessing this is idempotent.

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,9 +1,109 @@
...
+#### Properties
+
+- `source_name` - The name of the source to add. We'll use the resource's name if this isn't provided.
+- `source` - The source URL.
+- `bypass_proxy` - Whether or not to bypass the system's proxy settings to access the source.
+- `priority` - The priority level of the source.
+
+### powershell_package_source
+
+Use the `powershell_package_source` resource to register a powershell package repository
```
Comment (stuartpreston) [inline RELEASE_NOTES.md]: Missing period.

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -1,9 +1,109 @@
...
+
+### Actions
+
+- `register` - Registers and updates the powershell package source.
+- `unregister` - Unregisters the powershell package source.
+
+#### Properties
+
+- `source_name` - The name of the package source
```
Comment (stuartpreston) [inline RELEASE_NOTES.md]: Missing period.

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -1,9 +1,109 @@
...
+- `url` - The url to the package source
```
Comment (stuartpreston) [inline RELEASE_NOTES.md]: Missing period.

File: RELEASE_NOTES.md [hunk 5]
```diff
@@ -1,9 +1,109 @@
...
 
+# Chef Client Release Notes 14.3:
+
+## New Preview Resources Concept
+
+This release of Chef introduces the concept of Preview Resources. Preview resources behave the same as a standard resource built into Chef, except Chef will load a resource with the same name from a cookbook instead of the built-in preview resource.
+
+What does this mean for you? It means we can introduce new resources in Chef without breaking existing behavior in your infrastructure. For instance if you have a cookbook with a resource named `manage_everything` and a future version of Chef introduced a preview resource named `manage_everything` you will continue to receive the resource from your cookbook. That way outside of a major release your won't experience a potentially breaking behavior change from the newly included resource.
+
+Then when we perform our yearly major release we'll remove the preview designation from all resources, and the built in resources will take precedence over resources with the same names in cookbooks.
```
Comment (stuartpreston) [inline RELEASE_NOTES.md]: Where are the instructions for if you want the preview resource to override a resource in your cookbook? Possible?

Comment (tas50) [reply RELEASE_NOTES.md]: Doesn't do that. Not until 15

Comment (coderanger) [reply RELEASE_NOTES.md]: They way to get that is to remove the cookbook from your infra. But often easier said than done.

**Pull request merged**